 api02 Pulling 
 api01 Pulling 
 86f7c61081c3 Already exists 
 86f7c61081c3 Already exists 
 4f4fb700ef54 Already exists 
 4f4fb700ef54 Already exists 
 0ebfbbdde06b Already exists 
 0ebfbbdde06b Already exists 
 7cf3ade9cd7b Already exists 
 7cf3ade9cd7b Already exists 
 973648749997 Pulling fs layer 
 973648749997 Pulling fs layer 
 973648749997 Downloading [>                                                  ]  291.4kB/29.13MB
 973648749997 Downloading [>                                                  ]  291.4kB/29.13MB
 973648749997 Downloading [==>                                                ]  1.176MB/29.13MB
 973648749997 Downloading [==>                                                ]  1.176MB/29.13MB
 973648749997 Downloading [===>                                               ]  1.778MB/29.13MB
 973648749997 Downloading [===>                                               ]  1.778MB/29.13MB
 973648749997 Downloading [======>                                            ]  3.564MB/29.13MB
 973648749997 Downloading [======>                                            ]  3.564MB/29.13MB
 973648749997 Downloading [=======>                                           ]  4.453MB/29.13MB
 973648749997 Downloading [=======>                                           ]  4.453MB/29.13MB
 973648749997 Downloading [============>                                      ]  7.418MB/29.13MB
 973648749997 Downloading [============>                                      ]  7.418MB/29.13MB
 973648749997 Downloading [==================>                                ]  11.01MB/29.13MB
 973648749997 Downloading [==================>                                ]  11.01MB/29.13MB
 973648749997 Downloading [=========================>                         ]  14.84MB/29.13MB
 973648749997 Downloading [=========================>                         ]  14.84MB/29.13MB
 973648749997 Downloading [=============================>                     ]  17.22MB/29.13MB
 973648749997 Downloading [=============================>                     ]  17.22MB/29.13MB
 973648749997 Downloading [======================================>            ]   22.3MB/29.13MB
 973648749997 Downloading [======================================>            ]   22.3MB/29.13MB
 973648749997 Downloading [==============================================>    ]  27.36MB/29.13MB
 973648749997 Downloading [==============================================>    ]  27.36MB/29.13MB
 973648749997 Verifying Checksum 
 973648749997 Verifying Checksum 
 973648749997 Download complete 
 973648749997 Download complete 
 973648749997 Extracting [>                                                  ]  294.9kB/29.13MB
 973648749997 Extracting [>                                                  ]  294.9kB/29.13MB
 973648749997 Extracting [=========>                                         ]  5.603MB/29.13MB
 973648749997 Extracting [=========>                                         ]  5.603MB/29.13MB
 973648749997 Extracting [==================>                                ]  10.91MB/29.13MB
 973648749997 Extracting [==================>                                ]  10.91MB/29.13MB
 973648749997 Extracting [============================>                      ]  16.52MB/29.13MB
 973648749997 Extracting [============================>                      ]  16.52MB/29.13MB
 973648749997 Extracting [===================================>               ]  20.94MB/29.13MB
 973648749997 Extracting [===================================>               ]  20.94MB/29.13MB
 973648749997 Extracting [===============================================>   ]  27.72MB/29.13MB
 973648749997 Extracting [===============================================>   ]  27.72MB/29.13MB
 973648749997 Extracting [==================================================>]  29.13MB/29.13MB
 973648749997 Extracting [==================================================>]  29.13MB/29.13MB
 973648749997 Pull complete 
 973648749997 Pull complete 
 api02 Pulled 
 api01 Pulled 
 Container viniciusfcf-db-1  Creating
 Container viniciusfcf-db-1  Created
 Container viniciusfcf-api02-1  Creating
 Container viniciusfcf-api01-1  Creating
 Container viniciusfcf-api02-1  Created
 Container viniciusfcf-api01-1  Created
 Container viniciusfcf-nginx-1  Creating
 Container viniciusfcf-nginx-1  Created
Attaching to api01-1, api02-1, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
db-1     | running bootstrap script ... ok
db-1     | performing post-bootstrap initialization ... ok
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | waiting for server to start....2024-02-09 17:29:25.995 UTC [49] LOG:  starting PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-09 17:29:26.008 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-09 17:29:26.018 UTC [52] LOG:  database system was shut down at 2024-02-09 17:29:25 UTC
db-1     | 2024-02-09 17:29:26.027 UTC [49] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/script.sql
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | INSERT 0 5
db-1     | INSERT 0 5
db-1     | 
db-1     | 
db-1     | 2024-02-09 17:29:26.467 UTC [49] LOG:  received fast shutdown request
db-1     | waiting for server to shut down....2024-02-09 17:29:26.476 UTC [49] LOG:  aborting any active transactions
db-1     | 2024-02-09 17:29:26.494 UTC [49] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
db-1     | 2024-02-09 17:29:26.495 UTC [50] LOG:  shutting down
db-1     | 2024-02-09 17:29:26.497 UTC [50] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-09 17:29:26.778 UTC [50] LOG:  checkpoint complete: wrote 938 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.233 s, sync=0.037 s, total=0.284 s; sync files=309, longest=0.011 s, average=0.001 s; distance=4298 kB, estimate=4298 kB; lsn=0/191D260, redo lsn=0/191D260
db-1     | 2024-02-09 17:29:26.786 UTC [49] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-02-09 17:29:26.907 UTC [1] LOG:  starting PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-09 17:29:26.907 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-02-09 17:29:26.907 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-02-09 17:29:26.920 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-09 17:29:26.938 UTC [67] LOG:  database system was shut down at 2024-02-09 17:29:26 UTC
db-1     | 2024-02-09 17:29:26.947 UTC [1] LOG:  database system is ready to accept connections
db-1     | 2024-02-09 17:29:34.405 UTC [78] FATAL:  role "root" does not exist
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
api02-1  | __  ____  __  _____   ___  __ ____  ______ 
api02-1  |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api02-1  |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api02-1  | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api02-1  | 2024-02-09 17:29:35,116 ERROR [io.qua.run.Application] (main) Failed to start application (with profile [prod]): java.lang.RuntimeException: Failed to start quarkus
api02-1  | 	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
api02-1  | 	at io.quarkus.runtime.Application.start(Application.java:101)
api02-1  | 	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:111)
api02-1  | 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:71)
api02-1  | 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:44)
api02-1  | 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:124)
api02-1  | 	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
api02-1  | Caused by: jakarta.persistence.PersistenceException: [PersistenceUnit: <default>] Unable to build Hibernate SessionFactory
api02-1  | 	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:129)
api02-1  | 	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:89)
api02-1  | 	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
api02-1  | 	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
api02-1  | 	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
api02-1  | 	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
api02-1  | 	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
api02-1  | 	at java.base@21.0.2/java.lang.Thread.runWith(Thread.java:1596)
api02-1  | 	at java.base@21.0.2/java.lang.Thread.run(Thread.java:1583)
api02-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:833)
api02-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:211)
api02-1  | Caused by: org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing sequence [Cliente_SEQ]
api02-1  | 	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.validateSequence(AbstractSchemaValidator.java:182)
api02-1  | 	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.performValidation(AbstractSchemaValidator.java:113)
api02-1  | 	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.doValidation(AbstractSchemaValidator.java:75)
api02-1  | 	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:295)
api02-1  | 	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:145)
api02-1  | 	at java.base@21.0.2/java.util.HashMap.forEach(HashMap.java:1429)
api02-1  | 	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:142)
api02-1  | 	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
api02-1  | 	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
api02-1  | 	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:315)
api02-1  | 	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:87)
api02-1  | 	... 9 more
api02-1  | 
api01-1  | __  ____  __  _____   ___  __ ____  ______ 
api01-1  |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api01-1  |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api01-1  | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api01-1  | 2024-02-09 17:29:35,121 ERROR [io.qua.run.Application] (main) Failed to start application (with profile [prod]): java.lang.RuntimeException: Failed to start quarkus
api01-1  | 	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
api01-1  | 	at io.quarkus.runtime.Application.start(Application.java:101)
api01-1  | 	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:111)
api01-1  | 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:71)
api01-1  | 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:44)
api01-1  | 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:124)
api01-1  | 	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
api01-1  | Caused by: jakarta.persistence.PersistenceException: [PersistenceUnit: <default>] Unable to build Hibernate SessionFactory
api01-1  | 	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:129)
api01-1  | 	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:89)
api01-1  | 	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
api01-1  | 	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
api01-1  | 	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
api01-1  | 	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
api01-1  | 	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
api01-1  | 	at java.base@21.0.2/java.lang.Thread.runWith(Thread.java:1596)
api01-1  | 	at java.base@21.0.2/java.lang.Thread.run(Thread.java:1583)
api01-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:833)
api01-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:211)
api01-1  | Caused by: org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing sequence [Cliente_SEQ]
api01-1  | 	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.validateSequence(AbstractSchemaValidator.java:182)
api01-1  | 	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.performValidation(AbstractSchemaValidator.java:113)
api01-1  | 	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.doValidation(AbstractSchemaValidator.java:75)
api01-1  | 	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:295)
api01-1  | 	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:145)
api01-1  | 	at java.base@21.0.2/java.util.HashMap.forEach(HashMap.java:1429)
api01-1  | 	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:142)
api01-1  | 	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
api01-1  | 	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
api01-1  | 	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:315)
api01-1  | 	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:87)
api01-1  | 	... 9 more
api01-1  | 
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
api02-1 exited with code 1
api01-1 exited with code 1
nginx-1  | 2024/02/09 17:29:37 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://127.0.0.1:8081/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:37 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://127.0.0.1:8082/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:39 [error] 28#28: *4 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:41 [error] 28#28: *5 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:43 [error] 28#28: *6 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
db-1     | 2024-02-09 17:29:44.515 UTC [93] FATAL:  role "root" does not exist
nginx-1  | 2024/02/09 17:29:45 [error] 28#28: *7 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:47 [error] 28#28: *8 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:49 [error] 28#28: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://127.0.0.1:8082/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:49 [error] 28#28: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://127.0.0.1:8081/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:51 [error] 28#28: *12 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:53 [error] 28#28: *13 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
db-1     | 2024-02-09 17:29:54.700 UTC [102] FATAL:  role "root" does not exist
nginx-1  | 2024/02/09 17:29:55 [error] 28#28: *14 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:57 [error] 28#28: *15 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:29:59 [error] 28#28: *16 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:30:01 [error] 28#28: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://127.0.0.1:8082/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/09 17:30:01 [error] 28#28: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://127.0.0.1:8081/clientes/1/extrato", host: "localhost:9999"
nginx-1 exited with code 0
nginx-1 exited with code 0
db-1     | 2024-02-09 17:30:03.840 UTC [1] LOG:  received fast shutdown request
db-1     | 2024-02-09 17:30:03.842 UTC [1] LOG:  aborting any active transactions
db-1     | 2024-02-09 17:30:03.845 UTC [1] LOG:  background worker "logical replication launcher" (PID 70) exited with exit code 1
db-1     | 2024-02-09 17:30:03.846 UTC [65] LOG:  shutting down
db-1     | 2024-02-09 17:30:03.848 UTC [65] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-09 17:30:03.873 UTC [65] LOG:  checkpoint complete: wrote 50 buffers (0.3%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.009 s, sync=0.008 s, total=0.027 s; sync files=17, longest=0.006 s, average=0.001 s; distance=238 kB, estimate=238 kB; lsn=0/1958DF8, redo lsn=0/1958DF8
db-1     | 2024-02-09 17:30:03.879 UTC [1] LOG:  database system is shut down
db-1 exited with code 0
